go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gardener/etcd-druid
 
-go 1.18
+go 1.19
```

Comment:
Do we want to change go.mod as well ? I think this will force dev to move to go 1.19 as well right ?

Reply:
I would recommend keeping the golang version same across CI build configuration and go.mod. There is no reason not to upgrade the dev environment to go 1.19

Reply:
go `v1.19` brings many fixes, especially in this PR's case, for the golangci-linter. Again, as mentioned above, it also makes sense to move to go `v1.19` for security reasons.